chapter_gpu_schedules/matmul.md

```diff
@@ -30,29 +30,30 @@ def matmul_timer_mxnet(n, ctx):
     return timer.timeit
 ```
 
-Compute the GFLOPS.
+Then we compute its GFLOPS and plot the performance baseline as a function of matrix size.
 
 ```{.python .input  n=37}
 sizes = 2**np.arange(8, 15, 1)
-times = [d2ltvm.bench_workload(matmul_timer_mxnet(int(n), 'gpu'))
+exe_times = [d2ltvm.bench_workload(matmul_timer_mxnet(int(n), 'gpu'))
          for n in sizes]
-mxnet_gflops = 2 * sizes **3 / 1e9 / np.array(times)
+mx_gflops = 2 * sizes ** 3 / 1e9 / np.array(exe_times)
```

Comment:
I merged too quickly without noticing the CI was failed. Can you fix this one? If you change from `mxnet_gflops` to `mx_gflops`, you need to update its later references.